src/components/SendModal/ProvisionIdentity/ProvisionIdentityForm/ProvisionIdentityForm.js

```diff
@@ -96,15 +97,22 @@ const ProvisionIdentityForm = (props) => {
                 if (provIdKey != null && idKey.data === provIdKey.data) {
                   assignedIdentity = identity.result.identity.identityaddress;
                 }
+                if (idKey.vdxfkey == primitives.ID_PARENT_VDXF_KEY.vdxfid &&
+                  idKey.data != sendModal.coinObj.system_id) {
+                   parentname = `.${identity.result.identity.name}.${sendModal.coinObj.display_ticker}@`
```

Comment:
I believe you have to use the fullyqualifiedname of the parent or risk leaving out some system or parent IDs